Fix:
arg_int.c:60:12: error: implicitly declaring library function 'isspace' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
arg_int.c:89:8: error: implicitly declaring library function 'toupper' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- src/arg_int.c.orig	2011-01-10 04:44:01.000000000 -0600
+++ src/arg_int.c	2021-03-05 08:37:44.000000000 -0600
@@ -24,6 +24,7 @@
 #include "config.h"
 #endif
 
+#include <ctype.h>
 /* #ifdef HAVE_STDLIB_H */
 #include <stdlib.h>
 /* #endif */
